website/src/content/api-reference/commercetools-frontend-application-shell-connectors.mdx

```diff
@@ -0,0 +1,314 @@
+---
+title: "@/application-shell-connectors"
+---
+
+<Subtitle>
+
+Complementary tools for the [`@commercetools-frontend/application-shell`](/api-reference/commercetools-frontend-application-shell).
+
+</Subtitle>
+
+# Installation
+
+```
+yarn add @commercetools-frontend/application-shell-connectors
+
+# or
+
+npm --save install @commercetools-frontend/application-shell-connectors
+```
+
+Additionally install the peer dependencies (if not present)
+
+```
+yarn add react @apollo/client
+
+# or
+
+npm --save install react @apollo/client
+```
+
+# Application context
+
+This section describes different parts of information provided through the application context and how it can be accessed.
+
+## Application properties
+
+This is the list of the main properties you can access in the application context.
+
+### `environment`
+
+An object containing information about the environment of the Custom Application such as the [ID](/api-reference/application-config#envproductionapplicationid))] of the Custom Application or its [entry point](/api-reference/application-config#entrypointuripath).
+
+```json
+{
+  "environment": {
+    "applicationId": "ckw3vt1hv034901uzio4bkzc3:my-custom-application",
+    "applicationName": "My Custom Application",
+    "entryPointUriPath": "my-custom-application"
+  }
+}
+```
+
+### `user`
+
+Logged in user information.
+
+```json
+{
+  "user": {
+    "id": "3mj76c04-f910-4223-84e1-f97b0fe291c2",
+    "email": "john.doe@domain.com",
+    "firstName": "John",
+    "lastName": "Doe",
+    "businessRole": "Consultant",
+    "locale": "en-GB",
+    "timeZone": "Europe/Berlin",
+    "projects": [
+      { "key": "my-project-A", "name": "My Project A" },
+      { "key": "my-project-B", "name": "My Project B" }
+    ]
+  }
+}
+```
+
+
+### `project`
+
+Current selected project information.
+
+```json
+{
+  "project": {
+    "countries": ["DE", "US"],
+    "currencies": ["EUR", "USD"],
+    "key": "project-a",
+    "languages": ["de-DE", "en-US"],
+    "name": "Project A",
+    "ownerId": "3mj76c04-f910-4223-84e1-f97b0fe291c2",
+    "ownerName": "My Organization"
+  }
+}
+```
+
+### `dataLocale`
+
+This property contains the information about the selected locale that is used to render localized data in your application.
+You should use it as the selected language in components such as [LocalizedTextField](https://uikit.commercetools.com/?path=/story/components-fields--localizedtextfield).
+
+```json
+{
+  "dataLocale": "de"
+}
+```
```

Comment:
````suggestion
```

<Info>

This is not the value used for the Merchant Center language, which is based on the `user.locale`.

</Info>
````